quadro.cpp

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <math.h>
+
+struct coefs {
+    double a;
+    double b;
+    double c;
+};
+
+coefs quadro_input() {
+    coefs coefs;
+    coefs.a = 0;
```

Comment:
why initialized with zero? also, you can just write `coefs coefs = { /*some, initial, values*/ };`